cluster/userdata-master.yaml

```diff
@@ -363,13 +363,47 @@ write_files:
             - name: CLUSTER_ID
               value: {{WEBHOOK_ID}}
             - name: TOKENINFO_URL
-              value: https://info.services.auth.zalando.com/oauth2/tokeninfo
+              value: http://127.0.0.1:9021/oauth2/introspect
```

Comment:
We need it next to the API server at least for bootstrapping a cluster, but maybe a cluster wide tokeninfo also makes sense.